PF/Assignment1/TreeSort/TreeSortMain.java

```diff
@@ -0,0 +1,52 @@
+
+//Program to implement sorting using Binary Search tree
+import java.util.Scanner;
+
+//Class to input details from user and print the sorted array
+public class TreeSortMain {
+	public static void main(String args[]) {
+		Scanner scan = new Scanner(System.in);
+		TreeSortImplementation<Integer> treeSort = new TreeSortImplementation<Integer>();
+		int noOfElements;
+		// Input number of elements from user
+		System.out.println("Enter no. of elements");
+		noOfElements = scan.nextInt();
+		int arr[] = new int[noOfElements];
+		int result[] = new int[noOfElements];
+		// Input roll numbers from user
+		System.out.println("Enter all the roll numbers of students");
+		arr[0] = scan.nextInt();
+		Node<Integer> node = new Node<Integer>(arr[0]);
+		for (int i = 1; i < noOfElements; i++) {
+			int checkDuplicate = 0;
+			while (checkDuplicate == 0) {
+				arr[i] = scan.nextInt();
+				int count = 0;
```

Comment:
ACTION: Create a method to check duplicate value in array and also cover that in test case.